test/fixtures/box/labelDecoration.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+  tolerance: 0.0210,
+  config: {
+    type: 'scatter',
+    options: {
+      scales: {
+        x: {
+          display: false,
+          min: 0,
+          max: 10
+        },
+        y: {
+          display: false,
+          min: 0,
+          max: 10
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            box1: {
+              type: 'box',
+              xMin: 0.5,
+              xMax: 9.5,
+              yMin: 6.5,
+              yMax: 9.5,
+              backgroundColor: '#f5f5f5',
+              borderColor: 'black',
+              borderWidth: 1,
+              label: {
+                enabled: true,
+                content: ['text stroke width 1', 'text stroke color red'],
+                textStrokeWidth: 1,
+                textStrokeColor: 'red',
+                font: {
+                  size: 40
+                }
+              }
+            },
+            box2: {
+              type: 'box',
+              xMin: 0.5,
+              xMax: 9.5,
+              yMin: 2.5,
+              yMax: 6,
+              backgroundColor: '#f5f5f5',
+              borderColor: 'black',
+              borderWidth: 1,
+              label: {
+                enabled: true,
+                textStrokeWidth: 2,
+                color: 'white',
+                content: 'text stroke width 2',
+                font: {
+                  size: 40
+                },
+                position: {
+                  x: 'start',
+                  y: 'start'
+                }
```

Comment:
Spec is faster, but a fixture is ok too.

Reply:
as far as I have seen, the font remains the same, the width remain the same, even if the stroke width is passed. I was expecting the width will different, like the shapes. Therefore we could remove the `textStrokeWidth` from the cache key.

Reply:
I confirm that `lineWidth` set by `textStrokWidth` doesn't affect the `ctx.measureText(text)`. Sounds weird

Reply:
Maybe the actualBoundingBox* properties of the textMetrix returned change?
https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics

Reply:
I have the feeling that the stroke of the text is calculated inside the character. In fact, the below picture, is the result of chart (log provided above):

![textStrokeWidth](https://user-images.githubusercontent.com/11741250/153488403-87db9fef-b670-422c-9bd5-f3540aa24f12.png)

As you can see, the border seems really part of the font, not like the path where the border is added.
This will simplify the code a bit.

Reply:
well, the stroke is added OUTSIDE the character (https://stackoverflow.com/questions/13627111/drawing-text-with-an-outer-stroke-with-html5s-canvas) but the TextMetrics is not taking care. Anyway, I've checked and it's enough to the textStrokeWidth to the width from TextMetrics, and the same to the height.